source/math/FloatBox2D.ooc

```diff
@@ -63,6 +63,18 @@ FloatBox2D: cover {
 	shrink: func ~fraction (margin: Float) -> This {
 		this pad(-margin * this height / 2.0f)
 	}
+	resizeTo: func (size: FloatSize2D) -> This {
+		This createAround(this center, size)
+	}
+	scale: func (value: Float) -> This {
+		This createAround(this center, value * this size)
+	}
+	enlargeTo: func (size: FloatSize2D) -> This {
+		This createAround(this center, this size + ((size - this size) maximum~Float(0.0f)))
+	}
+	reduceTo: func (size: FloatSize2D) -> This {
```

Comment:
I maintain that "reduce" is the opposite of "increase", not the opposite of "enlarge". What's wrong with `shrinkTo`?